src/Map/types/viewerProperty.ts

```diff
@@ -0,0 +1,186 @@
+};
+
+export type TerrainProperty = {
+  enabled?: boolean;
+  type?: "cesium" | "arcgis" | "cesiumion";
+  url?: string;
+  normal?: boolean;
+  heightMap?: HeightMapProperty;
+};
+
+export type HeightMapProperty = {
+  type?: "custom";
+  colorLUT?: LUT;
+  minHeight?: number;
+  maxHeight?: number;
+  logarithmic?: boolean;
+};
+
+export type SceneProperty = {
+  backgroundColor?: string;
+  mode?: SceneMode;
+  verticalExaggeration?: number;
+  verticalExaggerationRelativeHeight?: number;
+  vr?: boolean;
+  light?: LightProperty;
+  shadow?: ShadowProperty;
+  imageBasedLighting?: ImageBasedLighting;
+};
+
+export type LightProperty = {
+  type?: "sunLight" | "directionalLight";
+  direction?: [x: number, y: number, z: number];
+  color?: string;
+  intensity?: number;
+};
+
+export type ShadowProperty = {
+  enabled?: boolean;
+  darkness?: number;
+  shadowMap?: ShadowMapProperty;
+};
+
+export type ShadowMapProperty = {
+  size?: 1024 | 2048 | 4096;
+  softShadows?: boolean;
+  darkness?: number;
+  maximumDistance?: number;
+};
+
+export type ImageBasedLighting = {
+  enabled?: boolean;
+  intensity?: number;
+  specularEnvironmentMaps?: string;
+  sphericalHarmonicCoefficients?: [number, number, number][];
+};
+
+export type TileProperty = {
+  id: string;
+  type?: string;
+  url?: string;
+  opacity?: number;
+  zoomLevel?: number[];
+  zoomLevelForURL?: number[];
+  heatmap?: boolean;
+};
+
+export type TileLabelProperty = {
+  id: string;
+  labelType: "japan_gsi_optimal_bvmap";
+  style: Record<string, any>;
+};
+
+export type SkyProperty = {
+  skyBox?: SkyBoxProperty;
+  sun?: SunProperty;
+  moon?: MoonProperty;
+  fog?: FogProperty;
+  skyAtmosphere?: SkyAtmosphereProperty;
+};
+
+export type SkyBoxProperty = {
+  show?: boolean;
+};
+
+export type SunProperty = {
+  show?: boolean;
+};
+
+export type MoonProperty = {
+  show?: boolean;
+};
+
+export type FogProperty = {
+  enabled?: boolean;
+  density?: number;
+};
+
+export type SkyAtmosphereProperty = {
+  show?: boolean;
+  lightIntensity?: number;
+  saturationShift?: number;
+  brightnessShift?: number;
+};
+
+export type CameraProperty = {
+  camera?: Camera;
+  allowEnterGround?: boolean;
+  limiter?: CameraLimiterProperty;
+};
+
+export type CameraLimiterProperty = {
+  enabled?: boolean;
+  targetArea?: Camera;
+  targetWidth?: number;
+  targetLength?: number;
+  showHelper?: boolean;
+};
+
+export type RenderPeropty = {
+  antialias?: "low" | "medium" | "high" | "extreme";
+  ambientOcclusion?: AmbientOcclusionProperty;
+};
+
+export type AmbientOcclusionProperty = {
+  enabled?: boolean;
+  quality?: "low" | "medium" | "high" | "extreme";
+  intensity?: number;
+  ambientOcclusionOnly?: boolean;
+};
+
+export type IndicatorProperty = {
+  type?: "default" | "crosshair" | "custom";
+  image?: string;
+  imageScale?: number;
+};
+
+export type AssetsProperty = {
+  cesium?: AssetsCesiumProperty;
+};
+
+export type AssetsCesiumProperty = {
```

Comment:
What's the difference of meta?

Reply:
I'm not quite sure, it was from exist code that we have access token from both meta and scene property. I'll try to remove this one and use meta only.
But for terrain ion accesstoken maybe we still need it?